README.md

```diff
@@ -17,12 +17,13 @@ Besides the admin-specific endpoints, there are differences between `api.github.
 Browsers
 </th><td width=100%>
 
-Load `@octokit/plugin-enterprise-server` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.pika.dev](https://cdn.pika.dev)
+Load `@octokit/plugin-enterprise-server` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.skypack.dev](https://cdn.skypack.dev)
+ 
```

Comment:
Can you remove this empty line? This is the reason why the tests are failing, we are checking all the files to follow certain formatting rules (using https://prettier.io/, you'll see that a lot in JavaScript project). And one rule is not to have two empty lines in Markdown (.md) files

Reply:
Yes I'd love to but how will I do that I cannot see any option and I am very new to GitHub

Reply:
no problem, happy to help! 

A simple way is to edit the file directly on github.com. In the "Files changed" tab, find the 3 dots on the top right, below the "review changes" dropdown

<img width="399" alt="image" src="https://user-images.githubusercontent.com/39992/97042872-8937ad00-1526-11eb-873d-430ee360dc63.png">

When you click "Edit file", then you get back to the editor mode, where you can make the change. Once you are done, press the green "Commit changes" button on the bottom of the page

Reply:
Hello, Thank you very much for your time and help. I made the change. Can you let me know if the change is fine now?

Reply:
It looks good now, thank you 👍🏼